source/_integrations/panel_iframe.markdown

```diff
@@ -24,9 +24,11 @@ panel_iframe:
   router:
     title: "Router"
     url: "http://192.168.1.1"
+    require_admin: true
```

Comment:
Please add this to the third example. We want our first example to always be as basic as possible.